meteor-app/imports/api/orders/orders.test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai'
+import { Meteor } from 'meteor/meteor'
+import Orders from '/imports/api/orders/schema'
+import '/imports/api/orders/methods'
+import Factory from '/imports/test/factories'
+import { resetDatabase } from '/imports/test/util-test'
+import { relativeTimeThreshold } from '../../../node_modules/moment';
+
+
+const goodOrder = Factory.build('order')
+const orderId = goodOrder._id
+
+if (Meteor.isServer) {
+  describe('testing orders method', () => {
+    it('success on inserting GOOD order', () => {
+      expect(() => Meteor.call('orders.insert', goodOrder)).to.not.throw()
+    })
+
+    it('success on removing an order', () => {
+      const order = Orders.findOne(orderId)
+      expect(() => Meteor.call('orders.remove', order)).to.not.throw()
+    })
+
+    it('success on updating an order', () => {
+      const orderedParts = goodOrder.orderedParts
+      expect(() => Meteor.call('orders.update', orderId, orderedParts)).to.not.throw()
```

Comment:
This no longer matches the parameters in orders.update (as per the other PR). It doesn't make sense to not put this file in the same PR - the tests should go with the source code they are testing

Reply:
Realized this after the other PR was submitted, have discussed this with the team and will be making some changes to the tests.
Is it okay to keep this PR open as the other PR has already been merged

Reply:
Please fix and push your changes and I can approve